platform/kube/inject/testdata/required.yaml.patch

```diff
@@ -1 +1 @@
-{"metadata":{"annotations":{"alpha.istio.io/sidecar":"injected(deprecated)","status.sidecar.istio.io":"injected-version-12345678"},"initializers":null},"spec":{"template":{"metadata":{"annotations":{"alpha.istio.io/sidecar":"injected(deprecated)","status.sidecar.istio.io":"injected-version-12345678"}},"spec":{"$setElementOrder/containers":[{"name":"hello"},{"name":"istio-proxy"}],"containers":[{"args":["proxy","sidecar","-v","2"],"env":[{"name":"POD_NAME","valueFrom":{"fieldRef":{"fieldPath":"metadata.name"}}},{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"fieldPath":"metadata.namespace"}}},{"name":"INSTANCE_IP","valueFrom":{"fieldRef":{"fieldPath":"status.podIP"}}}],"image":"docker.io/istio/proxy_debug:unittest","imagePullPolicy":"IfNotPresent","name":"istio-proxy","resources":{},"securityContext":{"readOnlyRootFilesystem":true,"runAsUser":1337},"volumeMounts":[{"mountPath":"/etc/istio/config","name":"istio-config","readOnly":true},{"mountPath":"/etc/istio/proxy","name":"istio-envoy"}]}],"initContainers":[{"args":["-p","15001","-u","1337"],"image":"docker.io/istio/proxy_init:unittest","imagePullPolicy":"IfNotPresent","name":"istio-init","resources":{},"securityContext":{"capabilities":{"add":["NET_ADMIN"]},"privileged":true}}],"volumes":[{"configMap":{"name":"istio"},"name":"istio-config"},{"emptyDir":{"medium":"Memory","sizeLimit":"0"},"name":"istio-envoy"}]}}}}
+{"metadata":{"annotations":{"alpha.istio.io/sidecar":"injected(deprecated)","status.sidecar.istio.io":"injected-version-12345678"},"initializers":null},"spec":{"template":{"metadata":{"annotations":{"alpha.istio.io/sidecar":"injected(deprecated)","status.sidecar.istio.io":"injected-version-12345678"}},"spec":{"$setElementOrder/containers":[{"name":"hello"},{"name":"istio-proxy"}],"containers":[{"args":["proxy","sidecar","-v","2"],"env":[{"name":"POD_NAME","valueFrom":{"fieldRef":{"fieldPath":"metadata.name"}}},{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"fieldPath":"metadata.namespace"}}},{"name":"INSTANCE_IP","valueFrom":{"fieldRef":{"fieldPath":"status.podIP"}}}],"image":"docker.io/istio/proxy_debug:unittest","imagePullPolicy":"IfNotPresent","name":"istio-proxy","resources":{},"securityContext":{"readOnlyRootFilesystem":true,"runAsUser":1337},"volumeMounts":[{"mountPath":"/etc/istio/config","name":"istio-config","readOnly":true},{"mountPath":"/etc/istio/proxy","name":"istio-envoy"},{"mountPath":"/etc/certs","name":"istio-certs","readOnly":true}]}],"initContainers":[{"args":["-p","15001","-u","1337"],"image":"docker.io/istio/proxy_init:unittest","imagePullPolicy":"IfNotPresent","name":"istio-init","resources":{},"securityContext":{"capabilities":{"add":["NET_ADMIN"]},"privileged":true}}],"volumes":[{"configMap":{"name":"istio"},"name":"istio-config"},{"emptyDir":{"medium":"Memory","sizeLimit":"0"},"name":"istio-envoy"},{"name":"istio-certs","secret":{"optional":true,"secretName":"istio.default"}}]}}}}
```

Comment:
this is impossible to read and understand, can we change this to multi line json so diffs make more sense.